client/src/components/LinkField/LinkField.js

```diff
@@ -396,12 +396,14 @@ const LinkField = ({
           onDragStart={handleDragStart}
           onDragEnd={handleDragEnd}
         >
-          <SortableContext
-            items={linkIDs}
-            strategy={verticalListSortingStrategy}
-          >
-            {links}
-          </SortableContext>
+          <ul className="link-picker__list">
+            <SortableContext
+              items={linkIDs}
+              strategy={verticalListSortingStrategy}
+            >
+              {links}
+            </SortableContext>
+          </ul>
```

Comment:
```suggestion
          <ol className="link-picker__list">
            <SortableContext
              items={linkIDs}
              strategy={verticalListSortingStrategy}
            >
              {links}
            </SortableContext>
          </ol>
```
Semantics is important here - and semantically since you can reorder the links and the order is important in the CMS context, `ol` should be used (regardless of the fact that in the frontend they're probably going to end up being in an unordered list)

Reply:
Updated